File: CS/Q182806/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace Q182806
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            // TODO: This line of code loads data into the 'issueListDataSet.Users' table. You can move, or remove it, as needed.
            this.usersTableAdapter.Fill(this.issueListDataSet.Users);
            itemsTableAdapter1.Fill(issueListDataSet.Items);
            projectsTableAdapter1.Fill(issueListDataSet.Projects);
            schedulerTableAdapter1.Fill(issueListDataSet.Scheduler);
        }
    }
}

File: VB/Q182806/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace Q182806
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			' TODO: This line of code loads data into the 'issueListDataSet.Users' table. You can move, or remove it, as needed.
			Me.usersTableAdapter.Fill(Me.issueListDataSet.Users)
			itemsTableAdapter1.Fill(issueListDataSet.Items)
			projectsTableAdapter1.Fill(issueListDataSet.Projects)
			schedulerTableAdapter1.Fill(issueListDataSet.Scheduler)
		End Sub
	End Class
End Namespace